.github/workflows/ci.yml

```diff
@@ -9,6 +9,7 @@ name: CI
 env:
   RUSTFLAGS: -Dwarnings
   RUST_BACKTRACE: 1
+  CARGO_UNSTABLE_SPARSE_REGISTRY: true
```

Comment:
Thanks for the explanation. Could you leave a TODO comment because I would like to remove that environment variable when the nightly toolchain is updated?

Reply:
DONE